### ISSUE TITLE
Resultados correctos de las pruebas 

### ISSUE THREAD

**Issue by ghost** [PDSW-PCVPMO/UP-TEHO#30]
Muchas pruebas tienen predefinido "assertTrue(true)" como resultado,Los resultados de las pruebas no son los esperados acorde a la especificación propuesta.

**Comment by danielospina-b** [comment]
@martinjhm271 Efectivamente, estas son solo pruebas de proximos sprints, dado que no se pueden correr ya que no estan implementados los metodos y fallaría CircleCI, se puso assertTrue y se les quito el tag @test
